docs/site/DataSources.md

```diff
@@ -12,7 +12,7 @@ A `DataSource` in LoopBack 4 is a named configuration for a Connector instance
 that represents data in an external system. The Connector is used by
 `legacy-juggler-bridge` to power LoopBack 4 Repositories for Data operations.
 
-![Datasource diagram](imgs/datasource.png)
+<img src="imgs/datasource.png" alt="Datasource diagram" width="300">
```

Comment:
Should the image be aligned to the center?

What if we change the font/shape size in the original diagram instead of having the browser resizing it?

Reply:
I think we should highlight the `DataSource` box and make the rest half transparent or lower contrast.

Reply:
The font and shape size is small because I just do a screen cap of the overview diagram, but somehow the image got expanded.  
There seems to be problems in displaying the image when I'm testing it locally.  So I'm trying to resize the image itself.

Reply:
@raymondfeng , it seems like making the image at the center is harder than I thought. :-)
We can use the html snippet to specify the alignment, i.e.
```
<center>
  <img src="pages/en/lb4/imgs/datasource.png">
</center>
```
The image appears fine when testing locally, however, the image will appear to be broken if you view within loopback-next. 

I think it's better to be able to view it within github or within the loopback-next repo (i.e.  `![Datasource diagram](imgs/datasource.png)` )  than to make the image to be at the center.  But I don't have a very strong preference. WDYT?

Reply:
I can accept the left-aligned diagrams with simple markdown syntax.